### PR TITLE
Update/verify client generator in scripts

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

File: cmd/libs/go2idl/args/args.go [hunk 1]
```diff
@@ -37,10 +37,12 @@ import (
 // Default returns a defaulted GeneratorArgs. You may change the defaults
 // before calling AddFlags.
 func Default() *GeneratorArgs {
-	return &GeneratorArgs{
+	generatorArgs := &GeneratorArgs{
 		OutputBase:       DefaultSourceTree(),
 		GoHeaderFilePath: filepath.Join(DefaultSourceTree(), "k8s.io/kubernetes/hack/boilerplate/boilerplate.go.txt"),
 	}
+	generatorArgs.AddFlags(pflag.CommandLine)
```
Comment (caesarxuchao) [inline cmd/libs/go2idl/args/args.go]: What if a user doesn't use the `Default()` to generate args? So IMO calling the `AddFlags` in `Execute` is better.

Comment (caesarxuchao) [inline cmd/libs/go2idl/args/args.go]: On second thought, I think the argument can go either way. Users are supposed to call `Default()` to generate the args, and if they don't, probably they don't want to parse the flags either. So LGTM :)

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Pull request merged**